go.mod

```diff
@@ -17,14 +17,64 @@
 //
 module github.com/gocql/gocql
 
+go 1.21
+
 require (
-	github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932 // indirect
-	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
-	github.com/golang/snappy v0.0.3
+	github.com/golang/snappy v0.0.4
 	github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed
-	github.com/kr/pretty v0.1.0 // indirect
-	github.com/stretchr/testify v1.3.0 // indirect
+	github.com/testcontainers/testcontainers-go v0.32.0
```

Comment:
You probably want to use the latest `v0.33.0`. just in case.

Reply:
upgraded to v0.33.0